SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -428,6 +452,8 @@ bool FLocalDeploymentManager::IsServiceRunningAndInCorrectDirectory()
 	FString SpotProjectInfoArgs = TEXT("alpha service project-info --json");
 	FString SpotProjectInfoResult;
 	FString StdErr;
+	FString SpatialServiceStatusArgs = FString::Printf(TEXT("service status %s"), *GetDomainEnvironmentStr(bIsInChina));
```

Comment:
`0.6.3` uses spatiald whereas `0.7.1` uses spot, this may introduce defects as they don't take the same arguments.

@improbable-danny & @joshuahuburn ptal

Reply:
spot appears to have a --endpoint URL which I suspect we set to the same as what we had in the DeploymentManager 

platform.api.spatialoschina.com

Reply:
```suggestion
```

Reply:
We don't need these changes, spot alpha service project-info --json does not need changing here